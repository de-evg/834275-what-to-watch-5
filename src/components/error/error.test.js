import React from "react";
import renderer from "react-test-renderer";
import Error from "./error";

it(`Should Error render correctly`, () => {
  const tree = renderer
    .create(<Error />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
