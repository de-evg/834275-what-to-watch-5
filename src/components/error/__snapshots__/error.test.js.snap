// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Error render correctly 1`] = `
<div>
  Сервер не доступен
</div>
`;
